packages/amplify-provider-awscloudformation/lib/system-config-manager.js

```diff
@@ -55,7 +62,6 @@ function setProfile(awsConfig, profileName) {
   }
 
   fs.writeFileSync(credentialsFilePath, ini.stringify(credentials));
```

Comment:
```suggestion
  fs.writeFileSync(credentialFd, ini.stringify(credentials));
  fs.closeSync(credentialFd)
```

Reply:
fs.writeFileSync first opens the file for write, write to the file, and then closes the file, so you don't need to call closeSync after fs.writeFileSync.

Reply:
We should see if the user has permission to write these files and show proper error message as this could be one of the edge cases where user does not have permission to write the file